benchmarks/bench_ml.py

```diff
@@ -352,7 +353,7 @@ def run(n_calls=32, n_runs=1, save_traces=True, n_jobs=1):
     * `n_jobs`: int
         Number of different repeats of optimization to run in parallel.
     """
-    surrogates = [GaussianProcessRegressor]
+    surrogates = [RandomForestRegressor, ExtraTreesRegressor, GradientBoostingQuantileRegressor, GaussianProcessRegressor]
```

Comment:
You could remove the `RandomForestRegressor` since we use the ExtraTreesRegressor by default. This is to create a smoothing of the decision surface.